server/stt/signal_hooks.py

```diff
@@ -141,7 +166,13 @@ def before_content_published(_sender: Any, item: Dict[str, Any], updates: Dict[s
         logger.warning(f"Failed to get 'assignment_id' of coverage '{coverage_id}'")
         return
 
-    _link_assignment_and_content(assignment_id, coverage_id, item.get("guid"), True)
+    try:
+        _link_assignment_and_content(assignment_id, coverage_id, item.get("guid"), True)
+    except Exception as err:
+        logger.exception(err)
+        logger.error("Failed to link coverage assignment to content")
```

Comment:
would be good in general to combine those to avoid an extra report on sentry, something like:
```py
logger.exception("Failed to link coverage assignment to content (%s)", err)
```
should log all the info into single report